### DISCUSSION TITLE
Sponsoring for an organization

### DISCUSSION THREAD

**Discussion by bradumbaugh** [squidfunk/mkdocs-material#2839] [General]
Hi @squidfunk,

How should an organization to sponsor this theme? I've read [the docs](https://squidfunk.github.io/mkdocs-material/insiders/#how-to-become-a-sponsor), but I'm a little confused about the best way to do it. Is one way better?

1. Sponsor the theme _as_ the organization, email you to request access for a bot account that's a member of the organization, and then use the bot account to fork and share the repo with other members of the organization (privately)?

2. Sponsor the theme _as_ the bot account that's a member of the organization, and then fork the repo and share it with other members of the organization (privately)?

Maybe there's a reason that the second option wouldn't work that I'm not thinking of?

Thanks!

**Comment by squidfunk** [comment]
Great question! You can do either, but depending on your organization the one or the other option may be preferred:

1. If you sponsor as an organization, the sponsorship amount will be added to your GitHub bill with the same billing cycle you have set for the entire organization. If you pay for GitHub on a yearly basis, your sponsoring will also be on a yearly basis. That's okay for some organizations, but not for all, which is why ...

2. ... the documentation recommends setting up a dedicated user account (a bot, not tied to a particular individual), which is granted access to your organization and sponsor with this user account. You can then set a different billing cycle for this user and have a dedicated bill for the sponsoring alone. This might make attribution of cost easier in larger corporations.

Either way, the user that is granted access then forks the repository to your organization, thus granting all users of your organization access to Insiders. The fork will be private by default and, of course, should stay private. Furthermore, by sponsoring with a user account, granting access is done automatically and no email is necessary. If an organization sponsors, there's no canonical user for me to add (some have all-private memberships), which necessitates the email.

If the sponsorship tier you choose is eligible for logo placement, you should drop me an email nonetheless, because some organizations want their logo to appear on the documentation and some don't. I'll only add it if I get permission to do so.

**Reply by bradumbaugh** [reply]
Great. Thanks very much, @squidfunk!
